app/assets/javascripts/application.js

```diff
@@ -123,6 +123,7 @@ function onContentLoad(){
       if ($(this).data('on-complete')){
         window[$(this).data('on-complete')].call(null, this, status);
       }
+      $(document).trigger('ContentLoad');
```

Comment:
You could use `data-on-complete="onContentLoad"`, or even better create a function that contains the hooks you need. 
I'm afraid adding this here may cause problems on other pages. This was the original idea of contentLoad, however over time it has become quite bloated and might cause breakage if executed twice on some pages (for example, IIRC a `select2` that has been initialized breaks if you try to initialize it again).